### ISSUE TITLE
Formats?

### ISSUE THREAD

**Issue by metaphim428** [snipe/moresecure#1]
Several formats are good.

There's the standard blog format that could be used to discuss real-world examples and it was suggested to do it in the format of zine. 

I like the idea of a zine in that it makes the content seem less overwhelming and puts it into bite size chunks.  Having a standard blog format could also be good to include more time relevant information between issues of the zine.

**Comment by snipe** [comment]
Yeah, ultimately, I'd like to keep all of the main content in the source code (possibly using Jekyll to generate static pages, for example) in markdown format. This makes it easy for anyone (non-HTML people) to contribute content in a format that is accessible and well-known, and keeps content away from layout/etc.

We could also potentially have both, with a main site linking to a Medium blog, etc. Ultimately, I won't have time to update a blog (as evidenced by how infrequently I update my own blog(s)).

**Comment by snipe** [comment]
One issue that could come up with using a static page generator is localization. Eventually I'd want this to be accessible in many languages, and that gets a little trickier with static content.

**Comment by metaphim428** [comment]
Google has a Translate button that can dropped in for the HTML stuff, but the zine would need actual translation before "going to print".

Although, considering the content, it might be better to have actual translation on stuff so that it's clear people in other languages so that things don't get lost in translation.

**Comment by snipe** [comment]
Google translate is rough though, and it can be used as a last resort, but I'm more interested in having our own actual translations (for the reasons you mentioned).
